### PR TITLE
Fix zugferd article

### PR DESCRIPTION
**Pull request by andruhovski**

### CODE REVIEW

File: net/advanced-operations/zugferd/_index.md
Comment (AnHolub) [inline net/advanced-operations/zugferd/_index.md:1]: you haven’t forgotten about "attach -zugferd" via .net? Or I didn't see

File: java/advanced-operations/zugferd/_index.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Electronic invoicing, or e-invoicing, is a digital way of creating, sending, receiving, and processing an invoice document. It eliminates paper and improves efficiency and savings for businesses in a B2B context. E-invoicing benefits both buyers and sellers.
+
+As more and more companies adopt e-invoicing to gain its advantages, they need standards to make sure that e-invoices can be easily understood by any software, no matter how they are created.
+
+Among the various data formats for e-invoices, the ZUGFeRD standard is becoming more popular in German-speaking and EU countries.
+
+ZUGFeRD is a German standard for electronic invoicing. It stands for "Central User Guide of the Forum for Electronic Invoicing Germany".
+
+The format is a hybrid that mixes a PDF document that can be read by humans and a XML data file that can be read by machines. It enables the exchange of invoice information between businesses and public authorities in a consistent and compatible way.
```
Comment (AnHolub) [inline java/advanced-operations/zugferd/_index.md:24]: a XML data - an XML data

File: python-net/advanced-operations/zugferd/_index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+E-invoicing is the process of creating, sending, receiving, and processing an electronic invoice document. It is fully digital from start to finish and helps businesses save money and improve efficiency in a B2B context. Both buyers and sellers can benefit from e-invoicing.
+
+As technology advances, millions of companies are adopting e-invoicing to enjoy its advantages. However, they need standards to ensure that e-invoices can be easily read and understood by any software, regardless of how they are generated.
+
+Out of the many data formats for e-invoices, the ZUGFeRD standard has become increasingly established in German-speaking and EU countries.
+
+ZUGFeRD is a standard for electronic invoicing in Germany. It stands for "Zentraler User Guide des Forums elektronische Rechnung Deutschland", which means Central User Guide of the Forum for Electronic Invoicing Germany.
+
+The format is a hybrid that combines a PDF document that can be read by humans and a XML data file that can be read by machines. It allows the exchange of invoice information between businesses and public authorities in a uniform and interoperable way.
```
Comment (AnHolub) [inline python-net/advanced-operations/zugferd/_index.md:24]: a XML data - an XML data

File: net/advanced-operations/zugferd/_index.md
Comment (andruhovski) [reply net/advanced-operations/zugferd/_index.md:1]: It has been written earlier. What's wrong?

File: java/advanced-operations/zugferd/_index.md (hunk 1 above)
Comment (andruhovski) [reply java/advanced-operations/zugferd/_index.md]: Fixed

File: python-net/advanced-operations/zugferd/_index.md (hunk 2 above)
Comment (andruhovski) [reply python-net/advanced-operations/zugferd/_index.md]: Fixed

**Review by AnHolub — APPROVED** [review]

**Pull request merged**